lib/explorers/api-parameters.explorer.ts

```diff
@@ -319,16 +320,22 @@ const mapParametersTypes = parameters =>
       },
       negate(isUndefined)
     );
+
+    paramWithStringType =
```

Comment:
I think we should extract this logic elsewhere because this function is dangerously growing.

Reply:
ok :) let me know when fixed